src/course-outline/CourseOutline.test.jsx

```diff
@@ -152,6 +163,27 @@ describe('<CourseOutline />', () => {
     });
   });
 
+  it('handles course outline fetch api errors', async () => {
+    axiosMock
+      .onGet(getCourseOutlineIndexApiUrl(courseId))
+      .reply(500, 'some internal error');
+
+    const { findByText, findByRole } = render(<RootWrapper />);
+    expect(await findByText('"some internal error"')).toBeInTheDocument();
+    // check errors in store
+    expect(store.getState().courseOutline.errors).toEqual({
+      outlineIndexApi: {
+        data: '"some internal error"',
+        status: 500,
+        type: 'serverError',
+      },
+    });
+
+    const dismissBtn = await findByRole('button', { name: 'Dismiss' });
+    fireEvent.click(dismissBtn);
+    expect(store.getState().courseOutline.errors).toEqual({});
+  });
```

Comment:
The error alert should not be dismissible for the outline

Reply:
@KristinAoki Fixed!